common/src/test/java/gov/cms/ab2d/common/service/CacheServiceImplTest.java

```diff
@@ -76,6 +74,13 @@ void setUp() {
         dataSetup.createCoverage(coveragePeriod, coverageSearchEvent, DataSetup.createBeneId());
     }
 
+    @AfterEach
+    void teardDown() {
+        coverageRepo.deleteAll();
```

Comment:
Bravo!  It was a real red flag to me to see deleteAll() in setup methods.  It makes it very hard to track down where the leaks actually are.